hyperparam-controller/controller.py

```diff
@@ -0,0 +1,87 @@
+from pprint import pprint
+import itertools
+import json, yaml
+from kubernetes import client, config, watch
+
+
+group = "argoproj.io"
+version = "v1alpha1"
+plural = "hyperparamworkflows"
+
+
+config.load_incluster_config()
+
+
+namespace = 'default'
```

Comment:
Do we want to move this into an env variable or something? What if the tool will be used across multiple namespaces?

Reply:
yeah, we'll need to fix that, this is also in cronworkflow